lib/avo/configuration.rb

```diff
@@ -172,7 +174,9 @@ def license=(value)
     def license
       gems = Gem::Specification.map {|gem| gem.name}
 
-      @license ||= if gems.include?("avo-advanced")
+      @license ||= if gems.include?("avo-audit_logging")
+        "enterprise"
```

Comment:
This feature will not be bundled in a gem, so let's not force the license to enterprise.